user-guide/Troubleshooting/Procedures/Cloud_Connection_Issues.md

```diff
@@ -0,0 +1,75 @@
+---
+uid: Cloud_Connection_Issues
+---
+
+# Investigating dataminer.services connection issues
+
+This page provides solutions for potential connectivity problems with [dataminer.services](xref:AboutCloudPlatform).
+
+## Remote access issues
+
+If you encounter issues with dataminer.services features that may be caused by connection issues, follow the actions below in the specified order:
+
+1. [Check the status of the DxMs](#check-the-status-of-the-dxms).
+1. [Check the status of the DMS](#check-the-status-of-the-dms).
+1. [Check the cloud session](#check-the-cloud-session)
+1. [Make sure requirements are met](#make-sure-requirements-are-met)
+
+### Check the status of the DxMs
+
+A common cause for issues with dataminer.services features is a problem with the connection from the [DxMs](xref:DataMinerExtensionModules) towards dataminer.services.
+
+You can resolve this as follows:
+
+1. Verify whether all installed [DxMs](xref:DataMinerExtensionModules) are running, otherwise start them.
+
+1. Make sure the most recent versions of the DxMs are installed. See [Upgrading nodes to the latest DxM versions](xref:Managing_cloud-connected_nodes#upgrading-nodes-to-the-latest-dxm-versions).
+
+   This way, you avoid dealing with issues that have already been fixed.
+
+1. Verify the accessibility of the DMA host:
+
+   1. In the `C:\Skyline DataMiner\` folder of the DMA, open the file *MaintenanceSettings.xml*.
+
+   1. Look up the address specified in the *name* attribute of the *HTTPS* tag. For example:
+
+       ```xml
+       <MaintenanceSettings>
+          ...
+          <HTTPS enabled="true" name="foo.skyline.local"/>
+          ...
+       </MaintenanceSettings>
+       ```
+
+   1. Open a browser and navigate to this address.
+
+      If the page loads correctly, this indicates that the DMA host is locally accessible, but there may be an issue with the [proxy](xref:Connect_to_cloud_via_proxy) or [DMZ](xref:Connect_to_cloud_with_DMZ) settings.
+
+### Check the status of the DMS
+
+Verify the status of the DMS and ensure that the system is operational, if not, start the DMS.
+
+### Check the cloud session
+
+In DataMiner Cube, go to *System Center* > *Cloud* and check if the cloud identity is still valid.
```

Comment:
Why is "Renewing the cloud identity" kept in a separate section below? It would be more user-friendly if we immediately detailed this here. Also, if we do move this and remove the separate section below, is it still necessary to keep the "Remote access issues" subtitle (line 9)? Do you expect to add another category of issues later or is this the only category this page will ever contain?